manifests/compose.pp

```diff
@@ -0,0 +1,16 @@
+# docker::images
+class docker::compose {
+
+  exec{'Install composer':
+    path => '/usr/bin/',
+    cwd  => '/tmp',
+    command => 'curl -L https://github.com/docker/compose/releases/download/1.5.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose',
+    creates => '/usr/local/bin/docker-compose'
+  }->
+  file{'/usr/local/bin/docker-compose':
+    owner => 'root',
+    mode => '0766'
```

Comment:
Quite sure this should be '0755' otherwise it is writable by any user of the system and not executable.